cmd/root.go

```diff
@@ -305,10 +389,21 @@ func compareIssues(ghClient github.Client, jiraClient jira.Client) error {
 	jql := fmt.Sprintf("project='%s' AND cf[%s] in (%s)",
 		rootCmdCfg.GetString("jira-project"), ghIDFieldID, strings.Join(ids, ","))
 
-	jiraIssues, _, err := jiraClient.Issue.Search(jql, nil)
+	ji, res, err := makeJIRARequest(func() (interface{}, *jira.Response, error) {
+		return jiraClient.Issue.Search(jql, nil)
+	})
 	if err != nil {
+		log.Errorf("Error retrieving JIRA issues: %s", err)
+		defer res.Body.Close()
+		body, _ := ioutil.ReadAll(res.Body)
```

Comment:
do not suppress this error. In this particular case, if there was an error making the request, then there is a chance that res.Body is nil.